content/rs/administering/_index.md

```diff
@@ -5,10 +5,24 @@ weight: 50
 alwaysopen: false
 categories: ["RS"]
 ---
-This section covers everything you need to know to get up and running
-with RS, included installing, configuring a cluster, maintaining a
-cluster, creating databases, etc.
+This section covers everything you need to know to get up and running with RS.
 
-Topics include:
+{{% note %}}
+In addition to using the RS web UI to manage the cluster,
+you can also access cluster operations through the cluster REST API.
```

Comment:
Access? is that the correct word here? not- perform?

Reply:
@Paz-Yanover My intention was that you can get to the operations through the REST API, but I changed it to be more direct.